Unable to compile

I'm having issues getting the arduino code to compile.

I unzip the file, change the folder name to "bongo_cat" and load the bongo_cat.ino file. When I click the checkmark to verify, I get the following errors:

```
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:20,
                 from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:6:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\body\standardbody1.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:21:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\body\bodyeartwitch.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:24:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\faces\stock_face.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:25:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\faces\happy_face.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:26:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\faces\blink_face.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:27:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\faces\sleepy_face.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:30:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\paws\leftpawdown.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:31:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\paws\rightpawdown.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:32:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\paws\twopawsup.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:35:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\table\table1.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:38:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\effects\left_click_effect.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:39:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\effects\right_click_effect.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:40:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\effects\sleepy1.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:41:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\effects\sleepy2.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
In file included from C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations_sprites.h:42:
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\animations\effects\sleepy3.c:158:4: error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
  158 |   {LV_IMG_CF_RGB565A8, 0, 0, 64, 64},  // header: {cf, always_zero, reserved, w, h}
      |    ^~~~~~~~~~~~~~~~~~
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:35:8: error: 'lv_disp_draw_buf_t' does not name a type; did you mean 'lv_draw_buf_t'?
   35 | static lv_disp_draw_buf_t draw_buf;
      |        ^~~~~~~~~~~~~~~~~~
      |        lv_draw_buf_t
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:71:6: error: variable or field 'my_disp_flush' declared void
   71 | void my_disp_flush(lv_disp_drv_t *disp, const lv_area_t *area, lv_color_t *color_p) {
      |      ^~~~~~~~~~~~~
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:71:20: error: 'lv_disp_drv_t' was not declared in this scope; did you mean 'lv_fs_drv_t'?
   71 | void my_disp_flush(lv_disp_drv_t *disp, const lv_area_t *area, lv_color_t *color_p) {
      |                    ^~~~~~~~~~~~~
      |                    lv_fs_drv_t
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:71:35: error: 'disp' was not declared in this scope
   71 | void my_disp_flush(lv_disp_drv_t *disp, const lv_area_t *area, lv_color_t *color_p) {
      |                                   ^~~~
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:71:41: error: expected primary-expression before 'const'
   71 | void my_disp_flush(lv_disp_drv_t *disp, const lv_area_t *area, lv_color_t *color_p) {
      |                                         ^~~~~
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:71:75: error: expected primary-expression before '*' token
   71 | void my_disp_flush(lv_disp_drv_t *disp, const lv_area_t *area, lv_color_t *color_p) {
      |                                                                           ^
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:71:76: error: 'color_p' was not declared in this scope
   71 | void my_disp_flush(lv_disp_drv_t *disp, const lv_area_t *area, lv_color_t *color_p) {
      |                                                                            ^~~~~~~
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:71:6: error: variable or field 'my_disp_flush' declared void
   71 | void my_disp_flush(lv_disp_drv_t *disp, const lv_area_t *area, lv_color_t *color_p) {
      |      ^~~~~~~~~~~~~
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:71:20: error: 'lv_disp_drv_t' was not declared in this scope; did you mean 'lv_fs_drv_t'?
   71 | void my_disp_flush(lv_disp_drv_t *disp, const lv_area_t *area, lv_color_t *color_p) {
      |                    ^~~~~~~~~~~~~
      |                    lv_fs_drv_t
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:71:35: error: 'disp' was not declared in this scope
   71 | void my_disp_flush(lv_disp_drv_t *disp, const lv_area_t *area, lv_color_t *color_p) {
      |                                   ^~~~
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:71:41: error: expected primary-expression before 'const'
   71 | void my_disp_flush(lv_disp_drv_t *disp, const lv_area_t *area, lv_color_t *color_p) {
      |                                         ^~~~~
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:71:75: error: expected primary-expression before '*' token
   71 | void my_disp_flush(lv_disp_drv_t *disp, const lv_area_t *area, lv_color_t *color_p) {
      |                                                                           ^
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:71:76: error: 'color_p' was not declared in this scope
   71 | void my_disp_flush(lv_disp_drv_t *disp, const lv_area_t *area, lv_color_t *color_p) {
      |                                                                            ^~~~~~~
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino: In function 'void sprite_render_layers(sprite_manager_t*, lv_obj_t*, uint32_t)':
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:762:5: error: 'lv_draw_img_dsc_t' was not declared in this scope; did you mean 'lv_draw_image_dsc_t'?
  762 |     lv_draw_img_dsc_t img_dsc;
      |     ^~~~~~~~~~~~~~~~~
      |     lv_draw_image_dsc_t
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:763:27: error: 'img_dsc' was not declared in this scope
  763 |     lv_draw_img_dsc_init(&img_dsc);  // Initialize once outside loop
      |                           ^~~~~~~
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:763:5: error: 'lv_draw_img_dsc_init' was not declared in this scope; did you mean 'lv_draw_image_dsc_init'?
  763 |     lv_draw_img_dsc_init(&img_dsc);  // Initialize once outside loop
      |     ^~~~~~~~~~~~~~~~~~~~
      |     lv_draw_image_dsc_init
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:770:13: error: 'lv_canvas_draw_img' was not declared in this scope; did you mean 'lv_canvas_get_image'?
  770 |             lv_canvas_draw_img(canvas, 0, 0, sprite, &img_dsc);
      |             ^~~~~~~~~~~~~~~~~~
      |             lv_canvas_get_image
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino: In function 'void setup()':
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:809:28: error: 'draw_buf' was not declared in this scope
  809 |     lv_disp_draw_buf_init(&draw_buf, buf, NULL, SCREEN_WIDTH * 10);
      |                            ^~~~~~~~
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:809:5: error: 'lv_disp_draw_buf_init' was not declared in this scope; did you mean 'lv_draw_buf_init'?
  809 |     lv_disp_draw_buf_init(&draw_buf, buf, NULL, SCREEN_WIDTH * 10);
      |     ^~~~~~~~~~~~~~~~~~~~~
      |     lv_draw_buf_init
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:811:12: error: 'lv_disp_drv_t' does not name a type; did you mean 'lv_fs_drv_t'?
  811 |     static lv_disp_drv_t disp_drv;
      |            ^~~~~~~~~~~~~
      |            lv_fs_drv_t
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:812:23: error: 'disp_drv' was not declared in this scope
  812 |     lv_disp_drv_init(&disp_drv);
      |                       ^~~~~~~~
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:812:5: error: 'lv_disp_drv_init' was not declared in this scope; did you mean 'lv_fs_drv_init'?
  812 |     lv_disp_drv_init(&disp_drv);
      |     ^~~~~~~~~~~~~~~~
      |     lv_fs_drv_init
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:815:25: error: 'my_disp_flush' was not declared in this scope
  815 |     disp_drv.flush_cb = my_disp_flush;
      |                         ^~~~~~~~~~~~~
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:817:5: error: 'lv_disp_drv_register' was not declared in this scope; did you mean 'lv_fs_drv_register'?
  817 |     lv_disp_drv_register(&disp_drv);
      |     ^~~~~~~~~~~~~~~~~~~~
      |     lv_fs_drv_register
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino: In function 'void createBongoCat()':
C:\Users\cosmi\OneDrive\Documents\Arduino\bongo_cat\bongo_cat.ino:841:70: error: 'LV_IMG_CF_TRUE_COLOR' was not declared in this scope
  841 |     lv_canvas_set_buffer(cat_canvas, canvas_buf, CAT_SIZE, CAT_SIZE, LV_IMG_CF_TRUE_COLOR);
      |                                                                      ^~~~~~~~~~~~~~~~~~~~
exit status 1

Compilation error: 'LV_IMG_CF_RGB565A8' was not declared in this scope
```

I'm using lvgl 9.3.0. I tried several 8.x versions, but either got the same results or a sketch too big error.

Any suggestions?

--- a/README.md
+++ b/README.md
@@ -172,6 +172,7 @@ The `lv_conf.h` file configures the LVGL graphics library:
 2. **ESP32 Firmware**
    - Open `bongo_cat.ino` in Arduino IDE
    - Ensure libraries are installed
+   - Set Partition Scheme to `Huge APP (3MB No OTA/1MB SPIFFS)` under `Tools`
    - Upload to your ESP32
 
 3. **Desktop Application**

